pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/PublisherStats.java

```diff
@@ -44,6 +44,7 @@ public interface PublisherStats {
     long getProducerId();
 
     /** Whether partial producer is supported at client. */
+    @Deprecated
     boolean isSupportsPartialProducer();
```

Comment:
Why not remove this ?

Reply:
This is already exposed to the client interface. Are we allowed to make a breaking change?

Reply:
I think this is the correct change. If it has already been released, we cannot remove it without first deprecating it. I don't know that we have documented guidance on how long to keep methods like this marked as deprecated.